src/main/java/webs/hillel/ShoppingCartApp.java

```diff
@@ -0,0 +1,94 @@
+package webs.hillel;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.Scanner;
+
+@SpringBootApplication
+public class ShoppingCartApp implements CommandLineRunner {
+    private final ProductRepository productRepository;
+    private final Cart shoppingCart;
+
+    @Autowired
+    public ShoppingCartApp(ProductRepository productRepository, Cart shoppingCart) {
+        this.productRepository = productRepository;
+        this.shoppingCart = shoppingCart;
+    }
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ShoppingCartApp.class);
+
+    public static void main(String[] args) {
+        SpringApplication.run(ShoppingCartApp.class, args);
+    }
+
+    @Override
+    public void run(String... args) {
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        while (true) {
+            System.out.println("1. Add product to cart");
+            System.out.println("2. Remove product from cart");
+            System.out.println("3. View cart");
+            System.out.println("4. Exit");
```

Comment:
All these messages can be composed into a single one using a text block and put in 1 logger.info() call